index.js

```diff
@@ -1,5 +1,57 @@
-$(document).ready(function () {
-  $(".dropdown-btn").click(function () {
-    $(".dropdown-content").toggle("close");
-  });
+const MobileNav = document.querySelector('.mobile-nav');
+const NavbarContent = document.querySelector('.navbar-content');
+const MobileNavImg = document.querySelector('.mobile-nav-img');
+
+const IconClose = document.querySelector('.icon-close');
+
+const NavbarOptionsPortfolio = document.querySelector(
+  '.navbar-options-portfolio',
+);
+const NavbarOptionsAbout = document.querySelector('.navbar-options-about');
+const NavbarOptionsContact = document.querySelector('.navbar-options-contact');
+
+const HeadlineWrapper = document.querySelector('.headline');
+const logo = document.querySelector('.logo');
+// const header = document.querySelector("header");
+
+function blurbackground() {
+  HeadlineWrapper.style.filter = 'blur(6px)';
+  logo.style.filter = 'blur(6px)';
+}
+function undoblurbackground() {
+  HeadlineWrapper.style.filter = 'none';
+  logo.style.filter = 'none';
+}
+
+MobileNav.addEventListener('click', () => {
+  NavbarContent.classList.toggle('hide');
+  MobileNavImg.classList.toggle('hide');
+  IconClose.classList.toggle('hide');
+  blurbackground();
+});
+
+IconClose.addEventListener('click', () => {
+  NavbarContent.classList.toggle('hide');
+  MobileNavImg.classList.toggle('hide');
+  IconClose.classList.toggle('hide');
+  undoblurbackground();
+});
+
+NavbarOptionsPortfolio.addEventListener('click', () => {
+  NavbarContent.classList.toggle('hide');
+  MobileNavImg.classList.toggle('hide');
+  IconClose.classList.toggle('hide');
+  undoblurbackground();
+});
+NavbarOptionsAbout.addEventListener('click', () => {
+  NavbarContent.classList.toggle('hide');
+  MobileNavImg.classList.toggle('hide');
+  IconClose.classList.toggle('hide');
+  undoblurbackground();
+});
+NavbarOptionsContact.addEventListener('click', () => {
+  NavbarContent.classList.toggle('hide');
+  MobileNavImg.classList.toggle('hide');
+  IconClose.classList.toggle('hide');
+  undoblurbackground();
 });
```

Comment:
- [ ] In accordance to best practises, kindly ensure that you avoid code repetition. 👍 This will make your app to be more efficient and light-weighted. ℹ️ This [short tutorial](https://www.tutorialstonight.com/javascript-queryselectorall-foreach) might be helpful I guess. **Kindly pay attention to the essence of `querySelectorAll`**